Gemfile

```
@@ -82,6 +82,31 @@ gem "sys-filesystem",                   "~>1.4.3"
 gem "terminal",                                              :require => false
 gem "wim_parser",                       "~>1.0",             :require => false
 
+# gems to resolve security issues
+if RUBY_VERSION < "3.0"
+# probably running ruby 2.7.6
+# CVE-2021-33621 fixed: ruby 2.7.7 - https://github.com/advisories/GHSA-vc47-6rqg-c7f5
+gem "cgi",  "~> 0.1.0.2"
```

I think we'd prefer something more like

```suggestion
gem "cgi",  "~> 0.1", ">= 0.1.0.2"
```

This way, we can leave open future updates while ensuring minimums.